themes/vocabulary_theme/templates/layout.html

```diff
@@ -119,10 +119,18 @@
               {% endfor %}
             </div>
           </div>
-          <a class="navbar-item {% if this.is_child_of('/cc-search') %} is-active{% endif%}"
-            href="{{ '/cc-search'|url }}">
-            CC Search
-          </a>
+          <div class="navbar-item has-dropdown is-hoverable">
+            <a class="navbar-link is-arrowless {% if this.is_child_of('/cc-search') %} is-active{% endif%}">
+              CC Search<i class="icon caret-down"></i></a>
+            <div class="navbar-dropdown">
+              {% for href, title in [ 
+                ['/cc-search', 'Overview & Roadmap'],
+                ['/contributing-code/cc-search', 'Contribution Guide'],
```

Comment:
Please use the new URL path for this.

Reply:
Ok sure.

Reply:
I have made the changs. Thanks a lot for your guidance @kgodey